src/xamarin-client/Adaptive.ReactiveTrader.Client.Domain/ReactiveTrader.cs

```diff
@@ -1,49 +1,37 @@
 ﻿using System;
 using System.Reactive.Linq;
-using Adaptive.ReactiveTrader.Client.Domain.Authorization;
 using Adaptive.ReactiveTrader.Client.Domain.Concurrency;
 using Adaptive.ReactiveTrader.Client.Domain.Instrumentation;
 using Adaptive.ReactiveTrader.Client.Domain.Models.Execution;
 using Adaptive.ReactiveTrader.Client.Domain.Models.Pricing;
 using Adaptive.ReactiveTrader.Client.Domain.Models.ReferenceData;
 using Adaptive.ReactiveTrader.Client.Domain.Repositories;
 using Adaptive.ReactiveTrader.Client.Domain.ServiceClients;
-using Adaptive.ReactiveTrader.Client.Domain.Transport;
 using Adaptive.ReactiveTrader.Client.Domain.Transport.Wamp;
 using Adaptive.ReactiveTrader.Shared.Logging;
 
 namespace Adaptive.ReactiveTrader.Client.Domain
 {
     public class ReactiveTrader : IReactiveTrader, IDisposable
     {
-        private ConnectionProvider _connectionProvider;
         private ILoggerFactory _loggerFactory;
         private ILog _log;
-        private IControlRepository _controlRepository;
         private WampServiceClientContainer _serviceClientContainer;
 
-        public void Initialize(string username, string[] servers, ILoggerFactory loggerFactory = null, string authToken = null) 
+        public void Initialize(string username, string[] servers, ILoggerFactory loggerFactory = null, string authToken = null)
         {
             _loggerFactory = loggerFactory ?? new DebugLoggerFactory();
             _log = _loggerFactory.Create(typeof(ReactiveTrader));
-            _connectionProvider = new ConnectionProvider(username, servers, _loggerFactory);
             var concurrencyService = new ConcurrencyService();
 
             _serviceClientContainer = new WampServiceClientContainer(servers[0], username, concurrencyService, _loggerFactory);
-            _serviceClientContainer.ConnectAsync().Wait();
+            _serviceClientContainer.ConnectAsync();
```

Comment:
Yeah ideally we'd await this, but might be a pain from the initialize call chain. Is this fire and forget good enough?